sklearn/linear_model/ridge.py

```diff
@@ -867,10 +871,12 @@ def fit(self, X, y, sample_weight=None):
             #fit_params = {'sample_weight' : sample_weight}
             fit_params = {}
             gs = GridSearchCV(Ridge(fit_intercept=self.fit_intercept),
-                              parameters, fit_params=fit_params, cv=self.cv)
+                              parameters, fit_params=fit_params, cv=self.cv,
+                              scoring = self.scoring)
```

Comment:
Sorry, but I'm still confused about what should I compare. Is comparison between outcome of RidgeCV with specified `cv` and outcome of directly using GridSearchCV? Or something else.